charts/opentelemetry-operator/values.yaml

```diff
@@ -140,6 +140,13 @@ manager:
   # Enable vertical pod autoscaler support for the manager
   verticalPodAutoscaler:
     enabled: false
+
+    # Recommender responsible for generating recommendation for the object.
+    # List should be empty (then the default recommender will generate the recommendation)
+    # or contain exactly one recommender.
+    # recommenders:
+    # - name: custom-recommender-performance
```

Comment:
Given the VPA's recommenders list can only have a single item does it make sense for us to expose this field as a list instead of just a string to set? i.e. if `len(recommender.name) > 0` set it, otherwise don't?